tests/github/resources/github_conf/repo_no_rules/branch_protection_rules.json

```diff
@@ -0,0 +1,4 @@
+{
+    "message": "Branch not protected",
```

Comment:
this is the response whit status code 404?

Reply:
yes.